6211 ТягунАР Лаб. 4

HLLDRGN opened the pull request:
Выполнил 4 ЛР с калькулятором, также сделал 3 ЛР (Визитку), как главную страницу, через которую можно перейти ко всем другим ЛР (В них же были добавлены кнопки навигации)

HLLDRGN committed:
Create .gitkeep

HLLDRGN committed:
6211 ТягунАР Лаб.1

Первая сдача лабораторной работы

HLLDRGN committed:
Create .gitkeep

HLLDRGN committed:
6211 Тягун АР Лаб.2/Вар.24

HLLDRGN committed:
Delete Лабораторная работа 2 directory

HLLDRGN committed:
Create .gitkeep

HLLDRGN committed:
6211 ТягунАР Лаб.2/Вар.24

HLLDRGN committed:
Изменение лабиринта

Изменил код и  css, чтобы лабиринт стал более похожим на лабиринт

HLLDRGN committed:
Исправление недочетов 1 ЛР

Форматировал название классов согласно БЭМ, а также сделал лабиринт более похожим на лабиринт

HLLDRGN committed:
Update laba2.html

Внес изменения согласно общим моментам с форматированием, а также отдельные корректировки с заданным про детей и родителей

HLLDRGN committed:
Update styles.css

Внесение изменений в CSS файл

HLLDRGN committed:
Добавлена 4-ая лабораторная работа, в каждой из прошлых ЛР дополнительно добавлены кнопки навигации

mxwrlld commented on лаба 4/lab4/calculator.html:
```diff
@@ -0,0 +1,287 @@
+            background-color: #5a000f;
+            color: #b6ae7f;
+        }
+    </style>
```
Дублирование стилей из styles.css

mxwrlld commented on лаба 4/lab4/calculator.html:
```diff
@@ -0,0 +1,287 @@
+            <p><strong>Примечание:</strong> Калькулятор обрабатывает ошибки деления на ноль и некорректный ввод данных.</p>
+        </div>
+        
+        <a href="main.html" class="back-link">← Вернуться на главную страницу</a>
```
Ссылка некорретная - main.html лежит на уровень выше

mxwrlld commented on лаба 4/lab4/calculator.html:
```diff
@@ -0,0 +1,287 @@
+                num2Input.value = "4";
+            });
+        });
+    </script>
```
Вынесите в отдельный JS - файл

mxwrlld commented on лаба 4/lab4/calculator.html:
```diff
@@ -0,0 +1,287 @@
+                    default:
+                        showError("Неизвестная операция");
+                        return;
+                }
```
Выделите в отдельную функцию выполнение вычислений

HLLDRGN committed:
Перенос скрита калькулятора в отдельные JS

Теперь файл calculator.js несет в себе основную функцию вычислений, а main.js - основную логику калькулятора

HLLDRGN committed:
Update calculator.html

Удален повтор из файла styles.css, ссылка на главную страницу теперь работает, к .html подключены .js

HLLDRGN commented:
Внес все правки

mxwrlld commented:
Зачтено!
